packages/core/core/src/TransformerRunner.js

```diff
@@ -70,26 +62,28 @@ class TransformerRunner {
     }
 
     let input: TransformerInput = {
-      filePath: file.filePath,
+      filePath: req.filePath,
       ast: null,
       code,
-      env
+      env: req.env
     };
 
     let context = {
+      type: path.extname(req.filePath).slice(1),
```

Comment:
Curious why we are slicing here?

Reply:
path.extname leaves the `.` at the start